### ISSUE TITLE
ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

### ISSUE THREAD

**Issue by ArulCkk** [JunkFood02/Seal#1523]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site.
- [X] I've verified that I'm running the [**latest version**](https://github.com/yt-dlp/yt-dlp/releases/latest) of yt-dlp.
- [X] I've verified that I'm running the latest [**stable version**](https://github.com/JunkFood02/Seal/releases/latest/) of Seal or any later [**preview versions**](https://github.com/JunkFood02/Seal/releases).
- [X] I've read the [**Contributing guidelines**](https://github.com/JunkFood02/Seal/blob/main/CONTRIBUTING.md) and [**Code Of Conduct.**](https://github.com/JunkFood02/Seal/blob/main/CODE_OF_CONDUCT.md)
- [X] I've checked that the site i'm trying to download from is in the [**Supported Sites**](https://github.com/yt-dlp/yt-dlp/blob/master/supportedsites.md) list from yt-dlp
- [X] I understand that the issue will be (ignored/closed) if I intentionally remove or skip any mandatory field.

### Describe the bug

ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

### To Reproduce

ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

### Error reports

ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

### Screenshots & Screen Records

ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

### Additional context

ERROR: unable to open for writing: [Errno 36] File name too long: "/data/user/0/com.junkfood.seal/cache/tmp/it's okay but it's hurts 🥺🥺  kukira kmu akan berthan lama ：( tpi ternyta？ 🤧  makasih , dan semngat ya cintaku 😌 kamu keren kamu hebat!! aku bakal dukung kmu !!  aku bakal nemnin kmu sampai last show nanti dan seterusnya !! love you @adeljkt48   #adeljkt #adeljkt48 #adeljkt48👽 #adeljkt48🥵 #jkt48adel #jkt48adel👽💛 #adelgrad #adelgradution #jkt48video #jkt48 #jkt48newera  #CapCut  [7377632678369168645].mp4.part"

**Comment by spftware** [comment]
looks like the filename is a little *long*, maybe try shortening the title? maybe reword it and try again..

**Comment by JunkFood02** [comment]
settings - general - restrict filenames

**Comment by seanmavley** [comment]
Now under Download section

![Screenshot_20241010_081932_Seal](https://github.com/user-attachments/assets/8d63fef7-b9ec-45be-9abf-588b7aa325f5)
